packages/rich-text-types/src/schemaConstraints.ts

```diff
@@ -59,7 +59,7 @@ export type ListItemBlockEnum =
 
 export type TableRowBlockEnum = BLOCKS.TABLE_CELL | BLOCKS.TABLE_HEADER_CELL;
 
-export type TableCellEnum = BLOCKS.PARAGRAPH | BLOCKS.UL_LIST | BLOCKS.OL_LIST;
+export type TableCellEnum = ListItemBlockEnum;
```

Comment:
- Nit: Can we have a dedicated enum here? it would be handy when adjusting the normalisation logic in the editor 
- You also need to adjust the entry for table cell in the `CONTAINERS` below